<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(19, 24, 42);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="registrationFrame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>1201</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 48, 79);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="nameRegLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameRegLine">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>610</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(16, 91, 151, 255), stop:1 rgba(44, 156, 234, 255));
color: white;
border-radius: 20px;
}

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 rgb(119, 88, 127), stop: 1 rgb(28, 96, 255)
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 rgb(17, 21, 255), stop: 1 #ddd
        );
    }</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
     <property name="icon">
      <iconset resource="detechRs.qrc">
       <normaloff>:/images/images/ri_login-circle-fill.png</normaloff>:/images/images/ri_login-circle-fill.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>660</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Already have an account?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>0</y>
       <width>601</width>
       <height>741</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="detechRs.qrc">:/images/images/login image.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>591</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(79, 112, 190, 1);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="accountReg">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(79, 112, 190, 1);</string>
     </property>
     <property name="text">
      <string>Account Registration</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameRegLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameRegLine">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailRegLine">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="emailRegLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>310</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordRegLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>410</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordRegLine">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>430</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="confirmPasswordRegLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>510</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Confirm Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirmPasswordRegLine">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>530</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 2px solid rgb(121, 121, 121);
color: rgb(255, 255, 255);
border-radius: 8px;
padding-left: 15px;
}

QLineEdit:hover{
	border: 2px solid white;
}

QLineEdit:focus{
	border: 2px solid white;
}</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginHere">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>660</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(45, 158, 236);
font: 700 9pt &quot;Segoe UI&quot;;
text-decoration: underline;
}

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 rgb(70, 70, 70), stop: 1 rgb(85, 170, 255)
        );
    }</string>
     </property>
     <property name="text">
      <string>Login Here</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="detechLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DETECH</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="detechRs.qrc"/>
 </resources>
 <connections/>
</ui>
